src/app/(main)/about/page.tsx

```diff
@@ -0,0 +1,183 @@
+        <Paragraph>The Respond app is a mission management tool used to communicate mission information and enable responders to sign in to missions so that mission leadership knows how many responders are enroute.</Paragraph>
+      </Section>
+    </>
+  );
+}
+
+function AboutHomePage() {
+  return (
+    <Section id="homepage" variant="h4" title="Home Page">
+      <Paragraph>The Home page provides an overview of recent, ongoing, and future activities.</Paragraph>
+
+      <Section variant="h6" title="My Activity">
+        <Paragraph>The &quot;My Activity&quot; section contains an abbreviated list of the activities that you are currently involved with. It will only be present if are currently signed into, or on standby for, an activity.</Paragraph>
+      </Section>
+
+      <Section variant="h6" title="Missions">
+        <Paragraph>The &quot;Missions&quot; section will contain a list of missions that are currently active, or which concluded within the last 3 days. At present there is no option to view missions that are older than 3 days.</Paragraph>
+      </Section>
+
+      <Section variant="h6" title="Events">
+        <Paragraph>The &quot;Events&quot; section contains a list of Trainings or other Events that are currently active, or which concluded within the last 3 days. At present there is no option to view Events that are older than 3 days.</Paragraph>
+      </Section>
+    </Section>
+  );
+}
+
+function AboutResponding() {
+  return (
+    <>
+      <Section id="respond" variant="h4" title="Responding to a Mission">
+        <Paragraph>The primary functionality of Respond is to enable members to sign in and report they are responding to a Mission, Training or other Event. It also provides a way for mission leadership to keep track of the status of responders and see their relevant qualifications. In the future, the timeline events will be used to automate roster entry into the database.</Paragraph>
+
+        <Section variant="h6" title="Signing Into an Activity">
+          <Paragraph>By default, the button in each Mission or Event will indicate the expected next action, based on your current status. For example, if you have not signed in yet, or if you are on standby, it will say “Sign In”. Clicking Sign In will sign you into the Mission. Once you are signed in, the button will switch to Sign Out. Clicking it again will sign you out of the mission. Some statuses have a secondary option which appear in a dropdown menu.</Paragraph>
+        </Section>
+
+        <Section variant="h6" title="Standby for a Future Activity">
+          <Paragraph>Respond is introducing a new feature that enables you to indicate that you are available for a current or future mission, but are not currently responding. Missions that are scheduled to start in the future will default to Stand By; you will not be able to sign into a mission that has not started. To cancel, you can choose Sign Out from the dropdown menu.</Paragraph>
```

Comment:
Given this is intended to be more like a manual (as opposed to release notes), I suggest deleting "Respond is introducing a new feature". At this point the feature isn't _particularly_ new, and it will only get older from here. :)